src/WorkOS.net/Services/MFA/_interfaces/EnrollSmsFactorOptions.cs

```diff
@@ -0,0 +1,18 @@
+namespace WorkOS
+{
+    using Newtonsoft.Json;
+
+    public sealed class EnrollSmsFactorOptions : EnrollFactorOptions
+    {
+        public EnrollSmsFactorOptions(string phoneNumber)
```

Comment:
Once we add the constructor to `EnrollFactorOptions`, we'll want to call it with the correct type:

```cs
public EnrollSmsFactorOptions(string phoneNumber) : base("sms")
{
    PhoneNumber = phoneNumber;
}
```